TombEngine/Game/savegame.cpp

```diff
@@ -1327,14 +1327,14 @@ bool SaveGame::Save(int slot)
 	return true;
 }
 
-void SaveGame::AddGameDirToSavePath(const std::string & dirLocation)
+void SaveGame::AddGameDirToSavePath(const std::string& dirLocation)
```

Comment:
I see both "dir" and "path" used interchangeably in this PR. Why not stick to one? Or is there a specific distinction here? 🤔

Reply:
Usually "path" is where it could be a file rather than a dir, but I used `AddGameDirToSavePath` here because it sounds clearer than `AddGameDirToSaveDir` which makes it sound like some sort of weird filesystem addition is going on.